client/js/tract-details.js

```diff
@@ -193,88 +207,78 @@ var trpanels = d3.select("#trackdetails").selectAll("svg").data(trackdata);
              .attr("y", h - 280)
              .attr("class", "plot_text")
              .style("text-anchor", "end")
-             .style("stroke", function(d){return d3colors[d.name-1];} )
-             .style("fill", function(d){return d3colors[d.name-1];} )
-             .text(function(d) { return tracks[d.name-1]; });
+             .style("stroke", function(d){return d3colors[d.key-1];} )
+             .style("fill", function(d){return d3colors[d.key-1];} )
+             .text(function(d) { return tracks[d.key-1]; });
 
 // associate tracksline with each subject
     var  tracklines = trpanels.selectAll(".tracks")
-        .data(function(d){ return d.subjects; })
+        .data(function(d){ return d.values; })
         .enter().append("g")
         .attr("class", "tracks")
-        .attr("id", function(d,i){return "Subject"+(i);})
-        .style("opacity", 0.5)
-        .style("stroke-width", "2.5px")
+        .attr("id", function (d, i) {
+            if (i >= sub_data.length) {
+                return "Mean" + (i - sub_data.length);
+            } else {
+                return "Subject" + (i);
+            }
+        })
+        .style("opacity", 0.2)
+        .style("stroke-width", "1px")
         .on("mouseover", mouseover)
         .on("mouseout", mouseout)
-        .on("click", onclick );
+        .on("click", onclick);
+
+    d3.selectAll("#Mean0")
+        .style("opacity", 0.99)
+        .style("stroke-width", "3.5px")
 
 
-        tracklines.append("path")
-            .attr("class", "line")
-            //.attr("id", function(d,i){return(i);})
-            .attr("d",  function(d ) { return  line(d); });
+    tracklines.append("path")
+        .attr("class", "line")
+        .attr("d", function (d) { return line(d.values); });
 
   function mouseover() {
 	  if (!brushing) {
-		  if (isDown) {
-			  if ($(this).css("opacity") == 0.5) {
-				  // uses the opacity of the row for selection and deselection
-				  d3.selectAll('#' + this.id)
-					  .transition()
-					  .duration(50)
-					  .style("opacity", 1)
-					  .style("stroke-width", "5px");
-			  } else {
-				  d3.selectAll('#' + this.id)
-					  .transition()
-					  .duration(50)
-					  .style("opacity", 0.5)
-					  .style("stroke-width", "2.5px");
-			  }
-		  } else {
-			  if ($(this).css("stroke-width") == "2.5px") {
-				  // uses the stroke-width of the line clicked on to determine
-				  // whether to turn the line on or off
-				  d3.selectAll('#' + this.id)
-					  .transition()
-					  .duration(50)
-					  .style("opacity", 1);
+              if ($(this).css("stroke-width") == "1px") {			//uses the stroke-width of the line clicked on to determine whether to turn the line on or off
+                  d3.selectAll('#' + this.id)
+                      //.transition()
+                      //.duration(50)
+                      .style("opacity", 1)
+                      .style("stroke-width", "2px");
 			  }
 		  }
 	  }
   }
```

Comment:
This change broke the brushing behavior. But I think the fix is simple. Here is one extra closing bracket which closes the ready function. It should be moved below.